frontend/mirage/factories/project.js

```diff
@@ -1,20 +1,11 @@
-import { Factory, faker, association } from 'ember-cli-mirage';
+import { Factory, faker } from 'ember-cli-mirage';
 
-export default Factory.extend({  
+export default Factory.extend({
   afterCreate(project, server) {
     server.create('public-schools-analysis', { project });
     server.create('transportation-analysis', { project });
     server.create('community-facilities-analysis', { project });
   },
-
-  dataPackage: association({
-    schemas: {
-      mappluto: {
-        carto: 'mappluto_18v2'
-      }
-    }
-  }),
-
```

Comment:
This is being removed because we want more flexibility with this association?

Reply:
Yes, and to encourage explicitly defining the dataPackage relationship within tests only when necessary.